examples/createTokenForRecurrentPayment.php

```diff
@@ -145,7 +146,7 @@
  * Credit Card CVV (Security Code)
  * Credit Card Owner
  */
-$card = new Card('4111111111111111', '12', 2016, 123, 'Card Owner Name');
+$card = new Card('4111111111111111', '12', 2030, 123, 'Card Owner Name');
```

Comment:
Regarding the year, the best would be to have it calculated dinamically, like the current year+1

Reply:
Good point, done :+1: